apollo-runtime/src/main/java/com/apollographql/apollo/ApolloClient.java

```diff
@@ -77,6 +78,7 @@ public static Builder builder() {
   private final CacheControl defaultCacheControl;
   private final CacheHeaders defaultCacheHeaders;
   private final ApolloLogger logger;
+  private final ApolloCallTracker callTracker = new ApolloCallTracker();
```

Comment:
I would probably update this class later to enable this feature https://github.com/apollographql/apollo-android/issues/452